### PR TITLE
Add sample rst document

### PR DESCRIPTION
**Pull request by NoB0**
Relates to this [issue](https://github.com/iai-group/guidelines/issues/38)

### CODE REVIEW

File: docs/rst_sample.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+IAI Project Template
+======================
```
Comment (kbalog) [inline docs/rst_sample.rst:2]: Should be the same length as the line above

File: docs/rst_sample.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+* Linting
+
+  - Flake8
+
+Installation
+------------
+
+Create conda environment
+""""""""""""""""""""""""
```
Comment (kbalog) [inline docs/rst_sample.rst:22]: Use `^^` instead (see https://github.com/iai-group/guidelines/pull/51)

File: docs/rst_sample.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
+IAI Project Template
+======================
+
+This is an example of rst document.
```
Comment (kbalog) [inline docs/rst_sample.rst:4]: Nit: ... example of _an_ rst document

File: docs/rst_sample.rst (hunk 1 above)
Comment (NoB0) [reply docs/rst_sample.rst:2]: Done.

File: docs/rst_sample.rst (hunk 3 above)
Comment (NoB0) [reply docs/rst_sample.rst:4]: Done.

File: docs/rst_sample.rst (hunk 2 above)
Comment (NoB0) [reply docs/rst_sample.rst:22]: Done.

File: docs/rst_sample.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Run the following command to install and activate the conda environment:
+
+.. code-block:: shell
+
+    conda env update -f environment.yaml
+    conda activate template
+
+
```
Comment (kbalog) [inline docs/rst_sample.rst:36]: Nit: remove last empty line

**Review by kbalog — APPROVED** [review]
LGTM

File: docs/rst_sample.rst (hunk 4 above)
Comment (NoB0) [reply docs/rst_sample.rst:36]: Done.

**Pull request merged**